sdk/storage/azure-storage-extensions/setup.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+from setuptools import setup, Extension
+
+PACKAGE_NAME = "azure-storage-extensions"
+PACKAGE_PPRINT_NAME = "Azure Storage Extensions"
+
+package_folder_path = PACKAGE_NAME.replace('-', '/')
+
+setup(
+    name=PACKAGE_NAME,
+    version="1.0.0b1",
+    include_package_data=True,
+    description=PACKAGE_PPRINT_NAME,
+    long_description=open('README.md', 'r').read(),
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='ascl@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-extensions',
+    keywords="azure, azure sdk",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
```

Comment:
Is this prod release? iirc I saw a beta tag but could be wrong

Reply:
Yeah, its a beta. Fixed